wgpu-core/src/command/render.rs

```diff
@@ -1549,9 +1568,9 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                         state.is_ready().map_pass_err(scope)?;
 
                         let stride = match indexed {
-                            false => 16,
-                            true => 20,
-                        };
+                            false => std::mem::size_of::<DrawIndirectBuffer>(),
```

Comment:
nit: use `mem::size_of` if possible.

Reply:
you mean as in importing `std::mem` so I can do `mem::size_of::<DrawIndirectBuffer>()` or is there some other Rust thing I don't know? :)

Reply:
Yeah that's it :)